cookbook/core/scheduled_workflows/lp_schedules.py

```diff
@@ -101,11 +101,11 @@ def positive_wf(name: str):
 # Activating a Schedule
 # #####################
 #
-# After initializing your launch plan, activate it so that the schedule runs.
+# After initializing your launch plan, activate the specific version of the launch plan so that the schedule runs.
 #
 # .. code-block:: bash
 #
-#   flytectl update launchplan -p flyteexamples -d development {{ name_of_lp }} --activate
+#   flytectl update launchplan -p flyteexamples -d development {{ name_of_lp }} --version <foo> --activate
```

Comment:
Can we also add a block for deactivating the schedule and also link to flytectl docs for updating the launchplan in both the places

Reply:
Sure